src/main/java/nextstep/subway/line/dto/LineRequest.java

```diff
@@ -19,6 +20,16 @@ public LineRequest(String name, String color, Long upStationId, Long downStation
         this.upStationId = upStationId;
         this.downStationId = downStationId;
         this.distance = distance;
+        this.overFare = 0;
```

Comment:
```suggestion
        this(name, color, upStationId, downStationId, distance, 0);
```
이렇게 해주면 조금더 보기 편하지 않을까요...?

Reply:
수정하였습니다. 추가로 매직넘버도 수정하였습니다 😄